src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,42 @@
 package core.basesyntax;
 
-class Pair {
+import java.util.Objects;
 
+class Pair<F, S> {
+    private F first;
+    private S second;
+
+    private Pair(F first, S second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public F getFirst() {
+        return first;
+    }
+
+    public S getSecond() {
+        return second;
+    }
+
+    public static <O, P> Pair of(O first, P second) {
```

Comment:
Generics should be same as in a class declaration

Reply:
Sure, sorry